pkg/telemetry/global.go

```diff
@@ -0,0 +1,124 @@
+package telemetry
```

Comment:
I think I'd simplify this changeset by removing this until there is a need for it

Reply:
My gut check says using the global will be easier from an implementation perspective, else I _think_ we'll need to plumb an instance of telemetry into each service that records telemetry. Though I ack we aren't at that point yet, as we are mainly looking to record just the node version and DID. Will leave it up to you: happy to remove this, happy to leave it around.